tests/view_tests/tests/test_debug.py

```diff
@@ -666,78 +647,96 @@ def verify_paranoid_email(self, view):
                 self.assertNotIn(v, body)
 
 
-@override_settings(ROOT_URLCONF='view_tests.urls')
 class ExceptionReporterFilterTests(ExceptionReportTestMixin, LoggingCaptureMixin, SimpleTestCase):
     """
     Ensure that sensitive information can be filtered out of error reports.
     Refs #14614.
     """
-    rf = RequestFactory()
+    client_class = SensitiveTestClient
+    urls = 'view_tests.urls'
```

Comment:
Bad conflict resolution -- urls attribute is deprected in favor of `@override_settings(ROOT_URLCONF='view_tests.urls')`